utils/htttp_methods.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+
+class HttpMethods:
+    headers = {"Content-Type": "application/json"}
+
+    @staticmethod
+    def get(url: str) -> requests.Response:
+        result = requests.get(url, headers=HttpMethods.headers)
+        return result
```

Comment:
Можно не задавать `result`, так как мы его применяем только в `return`.

```suggestion
    @staticmethod
    def get(url: str) -> requests.Response:
        return requests.get(url, headers=HttpMethods.headers)
```